src/BuildScriptGenerator/FinalStretchVersions.cs

```diff
@@ -4,9 +4,25 @@ namespace Microsoft.Oryx.BuildScriptGenerator.Common
 {
     public static class FinalStretchVersions
     {
+        public const string FinalStretchDotNetCore11SdkVersion = "1.1.14";
+        public const string FinalStretchDotNetCore21SdkVersion = "2.1.818";
+        public const string FinalStretchDotNetCore22SdkVersion = "2.2.207";
+        public const string FinalStretchNetCoreApp10 = "1.0.16";
+        public const string FinalStretchNetCoreApp11 = "1.1.13";
+        public const string FinalStretchNetCoreApp20 = "2.0.9";
+        public const string FinalStretchNetCoreApp21 = "2.1.30";
+        public const string FinalStretchAspnetCoreApp21 = "2.1.30";
+        public const string FinalStretchNetCoreApp22 = "2.2.8";
+        public const string FinalStretchDotNetCore30SdkVersion = "3.0.103";
+        public const string FinalStretchNetCoreApp30 = "3.0.3";
+        public const string FinalStretchAspnetCoreApp30 = "3.0.3";
         public const string FinalStretchDotNetCore31SdkVersion = "3.1.420";
```

Comment:
This was my bad for missing it, but would you be able to make this configure the default based on the debian flavor? https://github.com/microsoft/Oryx/blob/main/src/BuildScriptGenerator/DotNetCore/VersionProviders/DotNetCoreOnDiskVersionProvider.cs#L23

You can just have it accept a parameter of debianFlavor, which you can pull from the `BuildScriptGeneratorOptions` that exists wherever it gets invoked

Reply:
also for 3.1, it looks like there are some stretch tests in this file that should get updated https://github.com/microsoft/Oryx/blob/main/tests/Oryx.BuildImage.Tests/BaseImage/BaseImageTest.cs#L43

Reply:
Also, just realized that we have updated the stretch version of 3.1.26

@daniv-msft do you know how long we are going to be updating the stretch versions for? The whole purpose of this PR was to lock in a final stretch version so that we can safely stop updating the stretch patch versions, but since we have started uploading them again it makes this whole thing more work for us as a team since we would need to update these values as well whenever we add a new stretch patch version...

Reply:
As soon as Antares moves to a newer image we are ok to stop uploading stretch versions, right?